src/test/java/net/sf/jabref/logic/importer/fetcher/MrDLibFetcherTest.java

```diff
@@ -0,0 +1,40 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.util.List;
+
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+import net.sf.jabref.testutils.category.FetcherTests;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import static org.junit.Assert.assertEquals;
+
+@Category(FetcherTests.class)
+public class MrDLibFetcherTest {
+
+    private MrDLibFetcher fetcher;
+    private BibEntry bibEntry;
+
+    @Before
+    public void setUp() {
+        fetcher = new MrDLibFetcher("", "");
+        bibEntry = new BibEntry();
+        bibEntry.setField(FieldName.TITLE, "lernen");
+    }
+
+    @Test
+    public void testPerformSearch() throws FetcherException {
+        List<BibEntry> bibEntrys = fetcher.performSearch(bibEntry);
+        assertEquals(false, bibEntrys.isEmpty());
```

Comment:
In the importer I test for a proper list. Have a look at the maxTest(). There I am taking a response from the server an look afterwards if the fields are set correctly.
What could I test here else? Because the Servers response will be different every time.